pkgs/development/libraries/gmime/3.0.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, pkgconfig, glib, gpgme, zlib, libgpgerror, libidn, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  # TODO: libidn support, PGP/MIME support, S/MIME support
```

Comment:
Are you want to address the TODO items before the merge?

Reply:
Yes, probably should have said so in the description. From a birdview it seems to me that it should be working, all the dependencies are there at least. Do you see anything missing, or that could be taken away?

Reply:
According to configure this is already enabled:

```
  PGP/MIME support:     yes (via GpgME)
  S/MIME support:       yes (via GpgME)
  libidn support:       yes
```

Reply:
Saw that here too.